### ISSUE TITLE
Organization joining link not working on FOSSASIA Internship Program Form

### ISSUE THREAD

**Issue by saloniankita** [fossasia/fossasia.org#717]
In order to be eligible for FOSSASIA Internship Program, there are some prerequisites for the applicants. One such prerequisite is being a public member of the FOSSASIA GitHub organization. While checking it out, I found that the link attached to this option isn't working. I'm not sure whether this has been disabled or not. 
I have attached the images below.
![Screenshot 2020-05-20 00 41 28](https://user-images.githubusercontent.com/56873389/82368542-3b78bf00-9a33-11ea-9b50-76335f220811.png)
![Screenshot 2020-05-20 00 41 37](https://user-images.githubusercontent.com/56873389/82368546-3ddb1900-9a33-11ea-98f9-f9f119fb79d4.png)

**Comment by mohak852** [comment]
I have checked it . It is working fine

**Comment by aryan7778** [comment]
@mohak852 it is not working for me



![Screenshot (83)](https://user-images.githubusercontent.com/36998845/82747077-7cfec680-9db3-11ea-85fc-edd2715d7af6.png)

**Comment by saloniankita** [comment]
@mohak852 I checked the link again. It's not working.

**Comment by mohak852** [comment]
Actually you are writing. For me also it's not working and I wanna work on this issue.So please assign this issue to me

**Comment by sparsh-99** [comment]
@mohak852 @saloniankita Yes, she is correct. The 2nd link is not working. I think it is being revoked by the org as the submission deadline is gone.
